src/main/kotlin/com/example/flashcardbackend/deckgroup/DeckGroupService.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.flashcardbackend.deckgroup
+
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class DeckGroupService(val repository: DeckGroupRepository) {
+
+    @Transactional(readOnly = true)
+    fun findDeckGroups(): List<DeckGroupListItemDTO> = repository.findAll()
```

Comment:
The implementation of this service method should map list of domain objects into list of DeckGroupListItemDTO objects.

Reply:
thanks for pointing it out! will fix :)